### ISSUE TITLE
Merging two .ann files

### ISSUE THREAD

**Issue by prantikaC** [nlplab/brat#1386]
Hi, I have two separate annotated .txt files for which I have two separate .ann files. Now I want to merge those two text files. Naturally, I want to view the annotations of both those files in the merged file but merging two .ann files is not a possibility because of the token indexes and the text spans. Can anyone please guide me regarding this?

**Comment by ghost** [comment]
I do not think we have scripts for this, a patch would be welcome if someone wants to write it.
